### PR TITLE
bonnie_bundle to CQM Measure model conversion

### PR DESCRIPTION
**Pull request by hossenlopp**
- [x] ❗️ Update gemfile to point to master branch or released gem of cqm-models with new measure model before merge. https://github.com/projecttacoma/cqm-models/pull/64

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1538
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated
- [x] Update cqm-execution test fixtures using update_cqm_execution_fixtures.sh script from server-scripts repo

**Bonnie Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/cqm/converter/bonnie_measure.rb [hunk 1]
```diff
@@ -0,0 +1,161 @@
+# CQM Converter module for HDS models.
+module CQM::Converter
+  # CQM Converter class for HDS based records.
+  module BonnieMeasure
```
Comment (mayerm94) [inline lib/cqm/converter/bonnie_measure.rb]: Why is this a module instead of a class? Also, update the comment above this

Comment (hossenlopp) [reply lib/cqm/converter/bonnie_measure.rb]: There is a no infrastructure that needs to be set up before conversion. Just static methods are needed, therefore just a module.

File: lib/cqm/converter/bonnie_measure.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+          id: bonnie_stratification['id'],
+          statement: CQM::StatementReference.new(
+            library_name: cqm_measure.main_cql_library,
+            statement_name: get_cql_statement_for_bonnie_population_key(bonnie_measure.populations_cql_map, bonnie_stratification['STRAT'])
+          )
+        )
+      end
+
+      # add observation info
+      bonnie_measure.observations.each do |bonnie_observation|
```
Comment (jbradl11) [inline lib/cqm/converter/bonnie_measure.rb]: I think we need to first check if `observations` exists.

Comment (hossenlopp) [reply lib/cqm/converter/bonnie_measure.rb]: Added safe navigation operators.
